back/db/migrate/20240812115140_add_seven_linear_scale_labels_to_custom_fields.rb

```diff
@@ -0,0 +1,25 @@
+class AddSevenLinearScaleLabelsToCustomFields < ActiveRecord::Migration[7.0]
+  class StubCustomField < ApplicationRecord
+    self.table_name = 'custom_fields'
+  end
+
+  def change
+    add_column :custom_fields, :linear_scale_label_multiloc_n1, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n2, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n3, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n4, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n5, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n6, :jsonb, default: {}, null: false
+    add_column :custom_fields, :linear_scale_label_multiloc_n7, :jsonb, default: {}, null: false
```

Comment:
I think it's better to make the field names end in `_multiloc` (e.g. `linear_scale_label_1_multiloc`). It's a convention throughout the codebase, and there are some scripts and stuff depending on the `_multiloc` postfix to detect multilingual content.

Reply:
Good catch. Thanks!

I had preferred the numbers at the end to make reading them easier, but your point is well taken.